planning/behavior_path_planner/include/behavior_path_planner/scene_module/pull_over/shift_pull_over.hpp

```diff
@@ -36,25 +35,32 @@ class ShiftPullOver : public PullOverPlannerBase
   ShiftPullOver(
     rclcpp::Node & node, const PullOverParameters & parameters,
     const LaneDepartureChecker & lane_departure_checker,
-    const std::shared_ptr<OccupancyGridBasedCollisionDetector> occupancy_grid_map);
+    const std::shared_ptr<OccupancyGridBasedCollisionDetector> & occupancy_grid_map);
```

Comment:
Is it okay to use reference value of shared pointer?

Reply:
I think it is not needed, but clang-tidy suggests this, so I followed it.